### PR TITLE
feat: add support for make-it-rain command

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Add support for make-it-rain command to collectibles.

Motivation and Context
---
Good for testing as well.

How Has This Been Tested?
---
Manually.

### CODE REVIEW

File: applications/tari_collectibles/src-tauri/src/main.rs [hunk 1]
```diff
@@ -26,13 +30,93 @@ mod schema;
 mod status;
 mod storage;
 
+// make-it-rain <asset_public_key> <amount_per_transaction> <number_transactions> <destination address> <source address - optional>
+fn make_it_rain(args: Vec<&str>, state: &ConcurrentAppState) -> Result<(), ExitError> {
```
Comment (stringhandler) [inline applications/tari_collectibles/src-tauri/src/main.rs]: Rather use structopt or clap and use named parameters

**Review by stringhandler — COMMENTED** [review]
Happy so far, just use structopt or clap

File: applications/tari_collectibles/src-tauri/src/main.rs (hunk 1 above)
Comment (Cifko) [reply applications/tari_collectibles/src-tauri/src/main.rs]: The structopt is already in the ConfigBootstrap I'm using that. I've split the functionality to several functions for better readability.

**Comment by delta1** [comment]
Please resolve conflicts when you can @Cifko

**Comment by Cifko** [comment]
> Please resolve conflicts when you can @Cifko

Done.

**Review by sdbondi — APPROVED** [review]

**Pull request merged**